src/main/java/seedu/address/logic/commands/MeetCommand.java

```diff
@@ -36,10 +36,15 @@
             + "DATE (DD.MM.YYYY) START (HH:MM) END (HH:MM) PLACE\n"
             + "END must be after START on the same DATE.\n"
             + "Example: " + COMMAND_WORD + " 3 -add 18.05.2021 15:00 18:00 MRT";
+    public static final String MESSAGE_INVALID_DATE =
+            "DATE in meeting should be in dd.MM.yyyy format and should be valid";
```

Comment:
Do you think it would be good to give an example? Not everyone might understand what MM means.